agenta-web/src/lib/helpers/evaluate.ts

```diff
@@ -1,13 +1,22 @@
 import {HumanEvaluationListTableDataType} from "@/components/Evaluations/HumanEvaluationResult"
-import {Evaluation, GenericObject, Variant} from "../Types"
+import {Evaluation, EvaluationScenario, GenericObject, Variant} from "../Types"
 import {convertToCsv, downloadCsv} from "./fileManipulations"
 
-export const exportABTestingEvaluationData = (evaluation: Evaluation, rows: GenericObject[]) => {
+export const exportABTestingEvaluationData = (
+    evaluation: Evaluation,
+    scenarios: EvaluationScenario[],
+    rows: GenericObject[],
+) => {
     const exportRow = rows.map((data, ix) => {
+        const inputColumns = data.inputs.reduce(
```

Comment:
```suggestion
        const inputColumns = evaluation.testset.testsetChatColumn
            ? {Input: evaluation.testset.csvdata[ix]?.[evaluation.testset.testsetChatColumn]}
            : data.inputs.reduce(
```

To cater to chat apps